DataFormats/HcalCalibObjects/interface/HBHEDarkening.h

```diff
@@ -0,0 +1,71 @@
+#ifndef HcalCalibObjects_HBHEDarkening_h
+#define HcalCalibObjects_HBHEDarkening_h
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"

```

Comment:
I don't see FWCore/ParameterSet in the list of allowed dependencies in the DataFormats 
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Restrictions_on_the_package_defi